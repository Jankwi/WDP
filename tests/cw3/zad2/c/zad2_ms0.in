7
3 2 8 1 9 3 6
3
5 7 0