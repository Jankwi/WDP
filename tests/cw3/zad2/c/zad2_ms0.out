3
5
2
7
8
0
1
9
3
6
